ArjunaCore/arjuna/classes/com/arjuna/ats/internal/arjuna/tools/osb/mbean/ObjStoreBrowser.java

```diff
@@ -2,7 +2,7 @@
    Copyright The Narayana Authors
    SPDX-License-Identifier: Apache-2.0
  */
-package com.arjuna.ats.arjuna.tools.osb.mbean;
+package com.arjuna.ats.internal.arjuna.tools.osb.mbean;
```

Comment:
Maybe this one shouldn't be internal? As you found it is the way it seems to be instantiated in WildFly: https://github.com/wildfly/wildfly/blob/956cd9ad69a0b65b681d14ba76ca4ec79a634e6b/transactions/src/main/java/org/jboss/as/txn/service/ArjunaTransactionManagerService.java#L13

Reply:
I moved this class to an internal package as it was annotated with `@Deprecated`. Do you think we should move this class, @mmusgrov?